library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/PojoInnerJoin.java

```diff
@@ -41,6 +41,11 @@ public PojoInnerJoin(int num, Class<?> outClass, String... keys)
 
   }
 
+  public PojoInnerJoin(Class<?> outClass, String[] leftKeys, String[] rightKeys)
```

Comment:
As this new one is added now, but previous can;t be removed because of backward compatibility, you could mark the old ctor as Deprecated.